pages/Tables.js

```diff
@@ -53,80 +34,231 @@ export default () => {
+            </td>
+          </tr>
+        </table>
+      </section>
+
+      <section>
+        <h4>Behaviour</h4>
+        <h5>Controls</h5>
+        <p>
+          <b>Baseline elements:</b>
+          <ul>
+            <li>
+              <b>Column header</b> - Column headers are titles for the column
+              content.
+            </li>
+            <li>
+              <b>Table content</b> - Table content displays all the raw data. As
+              default tables use 25 rows per page.
+            </li>
+            <li>
+              <b>Table pagination</b> - Table pagination is a simple navigation
+              method that lets you split a huge amount of content while
+              indicating that more pages exist.
+            </li>
+          </ul>
+        </p>
+        <p>
+          {' '}
+          <b>Interactive elements:</b>
+          <ul>
+            <li>
+              <b>Row selection</b> - Allows users to select a row and perform an
+              action. When a row checkbox is selected, the row should display a
+              background color and the number rows currently selected.
+            </li>
+            <li>
+              <b>Bulk selections</b> - Allows users to select all of the rows on
+              the table, to select only the visible rows or to deselect all
+              rows.
+            </li>
+            <li>
+              <b>Sorting tool</b> - To help users sort information, a column can
+              display column sorting by default. For custom sorting, place a
+              sort button at the top of the table with the description of the
+              header name.
+            </li>
+            <li>
+              <b>Filtering tool</b> - Filtering tools allow users to limit the
+              data displayed within a long table. There are 2 ways of using
+              filters on a table.
+              <ul>
+                <li>
+                  <b>Column filtering</b> - A filter for each column located
+                  below the column header. Only search and single dropdown
+                  single selection are allowed.
+                </li>
+                <li>
+                  <b>Expandable filtering</b> - An expandable filter button
+                  located at the top of the table. Note that custom filtering
+                  has to be built from scratch.
+                </li>
+              </ul>
+            </li>
+          </ul>
+        </p>
+      </section>
+      <section>
+        <h4>Content</h4>
+        <ul>
+          <li>
+            Table content should be aligned left, unless it’s a status then the
+            data should be center aligned
+          </li>
+          <li>
+            Column headings should be in bold and always match the text
+            alignment of the data in the column
+          </li>
+          <li>
+            Columns usually start with the person’s name or with an aggregate
```

Comment:
```suggestion
            Columns usually start with the person’s name or with an aggregate
```

The `’` isn't realyl valid, you should change them to be either `'` or html escaped (`&apos;`)